sbi/inference/posteriors/mcmc_posterior.py

```diff
@@ -160,6 +165,7 @@ def sample(
         mcmc_parameters: Dict = {},
         mcmc_method: Optional[str] = None,
         sample_with: Optional[str] = None,
+        num_workers: Optional[int] = None,
         show_progress_bars: bool = True,
     ) -> Tensor:
         r"""Return samples from posterior distribution $p(\theta|x)$ with MCMC.
```

Comment:
Docstring for `num_workers` should be added, while pointing out what it defaults to (i.e., `num_workers` passed during init)

Reply:
docstrings have been moved to `__init__`